settings.gradle.kts

```diff
@@ -15,6 +15,8 @@ dependencyResolutionManagement {
         google()
         mavenCentral()
         maven { url = uri("https://jitpack.io") }
+        /* Add the following line */
+        maven { url = uri("https://sdk-download.airbridge.io/maven") }
     }
```

Comment:
Added Airbridge Maven repository to the project's repositories. This is required to download and use the Airbridge SDK. According to the Airbridge SDK documentation, this step is necessary for installing the SDK via Gradle.